index.html

```diff
@@ -17,5 +20,47 @@ <h1 class="title">
       </p>
     </div>
   </section>
-  </body>
-</html>
+
+  <section class="section">
+    <div class="container">
+      <div class="columns is-centered">
+        <div class="card">
+          <div class="card-content login">
+            <p class="title">
+              Already a member?
+            </p>
+            <p class="subtitle">
+              Sign in to continue.
+            </p>
+            <div class="field">
+              <label class="label">Email</label>
+              <div class="control">
+                <input class="input" type="text" placeholder="Email address">
+              </div>
+            </div>
+            <div class="field">
+              <label class="label">Password</label>
+              <div class="control">
+                <input class="input" type="email" placeholder="My password">
```

Comment:
This form should have only one input field and the type of input should be text and placeholder should be `@Username`